modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/SystemDataSets.tsx

```diff
@@ -7,9 +7,112 @@ import {FrontendDataSet} from '@liferay/frontend-data-set-web';
 import React from 'react';
 
 import '../css/DataSets.scss';
-import {FDS_DEFAULT_PROPS} from './utils/constants';
 
-const SystemDataSets = ({namespace}: {namespace: string}) => {
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import {openModal} from 'frontend-js-web';
+
+import {API_URL, FDS_DEFAULT_PROPS} from './utils/constants';
+
+interface ISystemDataSet {
+	additionalAPIURLParameters: string;
+	defaultItemsPerPage: number;
+	description: string;
+	name: string;
+	restApplication: string;
+	restEndpoint: string;
+	restSchema: string;
+	symbol: string;
+	title: string;
+}
+
+const SelectSystemDataSetModalContent = ({
+	closeModal,
+	loadData,
+	systemDataSets,
+}: {
+	closeModal: Function;
+	loadData: Function;
+	systemDataSets: Array<ISystemDataSet>;
+}) => {
+	return (
+		<div className="select-system-data-set-modal-content">
+			<ClayModal.Header>
+				{Liferay.Language.get('create-system-data-set-customization')}
+			</ClayModal.Header>
+
+			<ClayModal.Body>
+				<FrontendDataSet
+					{...FDS_DEFAULT_PROPS}
+					id="SystemDataSets"
```

Comment:
I removed it because it is not really necessary. There will only ever be one DSM portlet, and internally this is unique. Maybe if we one day make this FDS a system data set, we can then add namespace :smile:  This cleans up API, removing unused `namespace`.